docs/release-notes/2020.04-release-notes.md

```diff
@@ -0,0 +1,201 @@
+Installation images now don't deliver Python 2.7 interpreters or libraries, however some software still requires Python 2.7.
+We've moved to GCC 7 as the default compiler.
+
+## Desktop software and libraries
+
+* LibreOffice 6.4.3 was added. There are some issues with GTK 3 theme, so "gen" VCL plugin is used by default. You can experiment with GTK3 plugin, setting SAL_USE_VCLPLUGIN environment variable to gtk3 explicitly.
+* VirtualBox was updated to 6.1.6. There are some issues with direct and VNC consoles with non-EFI guests.
+* VLC was updated to 3.0.10
+* ntfsprogs were updated to 2017.3.23AR.5
+* 32-bit versions of basix xorg tools were dropped
+* hplip was updated to 3.19.12
+* rhythmbox was updated to 3.4.4
+* Gstreamer was updated to 1.16.2
+* Due to serious performance issues with totem>3.0.1, we can't update totem to the latest version. Switching to own totem fork allowed to import upstream patches and remove dependency on Python 2.7.
+* UPower was updated to 0.99.11
+* Mate Power Manager policy was adjusted to avoid suspending computer unless specifically overwritten by user due to issues with wakeup on modern hardware.
+* XScreensaver was updated to 5.44
+* Gnome Connection Manager was updated to 1.2.0
+* MiniDLNA was added
+* XChat was removed
+
+## Common tools and libraries
+
+* Caiman installer was switched to Python 3.5
+* PKG was updated to use rapidjson instead of simplejson for json processing which reduced memory consumption on operations with large package catalogues
+* DDU security issue, allowing local attacker to gain root privileges under some rare cases, was fixed
+* net-snmp was updated to 5.8
+* Sudo was updated to 1.8.31
+* mozilla-nspr libraries were updated to 4.25
+* SQLite was updated to 3.31.1
+* OpenConnect was updated to 8.05, vpnc-scripts to 20190606 version
+* GNU Screen was updated to 4.8.0
+* tmux was updated to 3.0a
+* nano was updated to 4.8
+
+## Development tools and libraries
+
+* GCC compilers were updated to 7.5.0, 8.4, 9.3. GCC 7 is the default oi-userland compiler now.
+* Python was updated to 2.7.18, 3.5.9, a lot of python modules was updated
+* Early PEP-538 variant was imported to Python 3.5 to fix IPS issues in C locale
+* Clang 9 was added
+* Guile was updated to 2.2.7
+* Golang was updated to 1.13.8, 1.12.17
+* OpenJDK was updated to 1.8.232, icedtea-web - to 1.8.3
+* Ruby was updated to 2.6.6
+* PHP was updated to 7.3.17
+* Git was updated to 2.25.4
+* Mercurial was updated to 5.3.2
+* Glade was updated to 3.22.2
+* GNU TLS 3 was updated to 3.5.19, GNU TLS 2 was removed
+* Automake was updated to 1.16
+* Glib was updated to 2.62
+* Binutils were updated to 2.34
+
+## Server software
+
+* PostgreSQL 12 was added, PostgreSQL 9.4 was removed, PostgreSQL 9.5-11 were updated to latest minor releases
+* Barman was updated to 2.9. Server configurations should be updated. At least `archiver = on` option should be added to old configurations. Look at /usr/share/barman for examples.
```

Comment:
It's probably better to add extra spaces between "-" in PostgresSQL versions: "9.5 - 11" else it will look like a minor revision of 9.5